examples/airline/airline.py

```diff
@@ -0,0 +1,19 @@
+from configs.agents import *
+from openai_runner.repl import run_demo_loop
+
+context_variables = {
+    "customer_context": """Here is what you know about the customer's details:
+1. CUSTOMER_ID: customer_12345
+2. NAME: John Doe
+3. PHONE_NUMBER: (123) 456-7890
+4. EMAIL: johndoe@example.com
+5. STATUS: Premium
+6. ACCOUNT_STATUS: Active
+7. BALANCE: $0.00
+8. LOCATION: 1234 Main St, San Francisco, CA 94123, USA
+""",
+    "flight_context": """The customer has an upcoming flight from LGA (Laguardia) in NYC to LAX in Los Angeles.
+The flight # is 1919. The flight departure date is 3pm ET, 5/21/2024.""",
+}
+if __name__ == "__main__":
+    run_demo_loop(triage_agent, context_variables=context_variables, debug=True)
```

Comment:
**Suggestion:** Ensure that the `triage_agent` is defined before it is used in the `run_demo_loop` function to avoid runtime errors. [possible issue, importance: 10]
```suggestion
triage_agent = Agent(
    name="Triage Agent",
    instructions=triage_instructions,
    functions=[transfer_to_flight_modification, transfer_to_lost_baggage],
)
if __name__ == "__main__":
    run_demo_loop(triage_agent, context_variables=context_variables, debug=True)
```